components/tooltip/README.md

```diff
@@ -163,15 +189,21 @@ The `d2l-tooltip-help` component is used to display additional information when
 <!-- docs: end donts -->
 <!-- docs: end best practices -->
 
-<!-- docs: demo live name:d2l-tooltip-help autoSize:false size:small -->
+<!-- docs: demo live name:d2l-tooltip-help autoSize:false size:medium -->
 ```html
 <script type="module">
-  import '@brightspace-ui/core/components/tooltip/tooltip-help.js';
+  import '@brightspace-ui/core/components/tooltip/tooltip-help.js'; <!-- docs: start hidden content --> window.addEventListener('load', function () {
+    setTimeout(function() {
+      var tooltip = document.querySelector('d2l-tooltip-help[text="Helpful label"]:not(#help-tooltip)').shadowRoot.querySelector('d2l-tooltip');
```

Comment:
Perhaps a bit overkill since I could just add an `id` and select that.
However, since _this_ code is hidden, whereas the below other code is shown on the rendered page, I figured it is both unnecessary and potentially a bit confusing to have this help tooltip have a random `id` on it. 

If anyone has any thoughts/disagrees, feel free to let me know.